app/recordtransfer/management/commands/verify_settings.py

```diff
@@ -0,0 +1,390 @@
+import logging
+import os
+import re
+
+from bagit import CHECKSUM_ALGOS
+from django.conf import settings
+from django.contrib.sites.models import Site
+from django.core.exceptions import ImproperlyConfigured
+from django.core.management.base import BaseCommand
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """Verify application settings defined in settings/docker_dev.py."""
```

Comment:
Sorry, one last thing:

```suggestion
    """Verify application settings defined in the DJANGO_SETTINGS_MODULE."""
```

Reply:
This makes sense.